nomenclature/processor/region.py

```diff
@@ -403,6 +408,43 @@ def _filter_dict_args(
         }
 
 
+def _aggregate_with_model_native(
+    model_df: IamDataFrame,
+    vars: Union[str, List[str]],
+    common_region: str,
+    constituent_regions: List[str],
+    **aggregation_kwargs,
+) -> IamDataFrame:
+
+    # Get all results for common_region
+    model_native = model_df.filter(region=common_region, variable=vars)
+    # Aggregate for comparison
+    if aggregation_kwargs:
+        aggregated = _aggregate_region(
+            model_df, vars, *[common_region, constituent_regions], **aggregation_kwargs
```

Comment:
Why not use as args directly?
```suggestion
            model_df, vars, common_region, constituent_regions, **aggregation_kwargs
```

Reply:
Good point, I've changed it to your suggestion further down of using `regions` directly.